locale/zh/messages.json

```diff
@@ -984,49 +984,49 @@
   "You can run Docker containers with both container runtimes. The Universal Container Runtime (UCR) is better supported in DC/OS. <0>More information</0>.": "您可以使用两个容器运行时运行 Docker 容器。DC/OS 支持通用容器运行时 (UCR)。<0>更多信息</0>.",
   "You do not have access to this service. Please contact your {0} administrator or see <0>security documentation</0> for more information.": "您无权访问此服务。请联系您的 {0} 管理员或查看 <0>安全文档，</0> 以获取更多信息。",
-  "You do not have permission to create a group.": "",
+  "You do not have permission to create a group.": "您没有创建组的权限。",
   "You have several protocol options. <0>More Information</0>.": "您有几个协议选项。<0>更多信息</0>.",
-  "You must specify a command, an argument or a container with an image": "",
+  "You must specify a command, an argument or a container with an image": "您必须指定一个命令、一个自变量或一个带镜像的容器",
   "You need at least one package repository with some packages to be able to install packages. For more <0>information on repositories</0>.": "您至少需要一个含有一些包的包存储库，才能安装包。获取更多 <0>有关存储库的信息</0>.",
-  "Your computer seems to be offline. We will keep trying to reconnect.": "",
+  "Your computer seems to be offline. We will keep trying to reconnect.": "您的计算机似乎处于脱机状态。我们将继续尝试重新连接。",
   "Zone": "区",
   "Zones": "区",
   "by {mesosBuildUser}": "{mesosBuildUser} 天",
   "deployment": "部署",
   "deployments": "部署",
   "did not match": "不匹配",
   "matched": "已匹配",
-  "minutes": "",
+  "minutes": "分",
   "partially matched": "部分匹配",
   "scaling each service to 0 instances": "将每个服务缩放到 0 个实例",
-  "seconds": "",
+  "seconds": "秒",
   "task": "任务",
   "updating the number of instances of each service": "更新每个服务的实例数",
   "will be deleted": "将被删除",
   "your service's requirements ({requestedValue}).": "您服务的要求 ({requestedValue})。",
   "your service's role ({requestedValue}).": "您服务的角色 ({requestedValue})。",
-  "{0, plural, one {# group has services not limited by quota. Update service roles to have quota enforced.} other {# groups have services not limited by quota. Update service roles to have quota enforced.}}": "",
-  "{0, plural, one {# instance running out of {instancesCount}} other {# instances running out of {instancesCount}}}": "",
-  "{0, plural, one {# service has no quota limit.} other {# services have no quota limit.}}": "",
-  "{0}": "",
+  "{0, plural, one {# group has services not limited by quota. Update service roles to have quota enforced.} other {# groups have services not limited by quota. Update service roles to have quota enforced.}}": "{0, plural, one {# 组的服务不受配额限制。更新服务角色以执行配额。} other {# 组的服务不受配额限制。更新服务角色以执行配额。}}",
+  "{0, plural, one {# instance running out of {instancesCount}} other {# instances running out of {instancesCount}}}": "{0, plural, one {# 实例将用完 {instancesCount}} other {# 实例将用完 {instancesCount}}}",
+  "{0, plural, one {# service has no quota limit.} other {# services have no quota limit.}}": "{0, plural, one {# 服务没有配额限制。} other {# 服务没有配额限制。}}",
+  "{0}": "{0}",
```

Comment:
what the heck? looks like someone put a variable in a `Trans` that has not been `i18nMark`ed which is likely a mistake.

Reply:
Yea, someone i18nMarked or trans a variable. which will never actually translate anything.